services/util/ConfigurationStore.cpp

```diff
@@ -206,7 +205,9 @@ namespace services
             ++operationId;
             writeRequested = false;
             writingBlob = true;
-            Serialize(*activeBlob, [this, thisId]() { inactiveBlob->Erase([this, thisId]() { BlobWriteDone(); NotifyObservers([thisId](ConfigurationStoreObserver& observer) { observer.OperationDone(thisId); }); }); });
+            Serialize(*activeBlob, [this, thisId]()
+                { inactiveBlob->Erase([this, thisId]()
+                      { BlobWriteDone(); NotifyObservers([thisId](ConfigurationStoreObserver& observer) { observer.OperationDone(thisId); }); }); });
```

Comment:
Why does the lambda in this line get to stay on the same line, while 14 lines lower the lambda is put on a new line?